src/modules/rtpengine/api.h

```diff
@@ -26,18 +26,23 @@
 
 #include "../../core/parser/msg_parser.h"
 
-typedef int (*rtpengine_start_recording_f)(
-		struct sip_msg *msg, char *str1, char *str2);
-typedef int (*rtpengine_answer_f)(struct sip_msg *msg, char *str1, char *str2);
-typedef int (*rtpengine_offer_f)(struct sip_msg *msg, char *str1, char *str2);
-typedef int (*rtpengine_delete_f)(struct sip_msg *msg, char *str1, char *str2);
+typedef int (*rtpengine_start_recording_f)(struct sip_msg *msg);
+typedef int (*rtpengine_answer_f)(struct sip_msg *msg, str *str);
+typedef int (*rtpengine_answer0_f)(struct sip_msg *msg);
+typedef int (*rtpengine_offer_f)(struct sip_msg *msg, str *str);
+typedef int (*rtpengine_offer0_f)(struct sip_msg *msg);
+typedef int (*rtpengine_delete_f)(struct sip_msg *msg, str *str);
+typedef int (*rtpengine_delete0_f)(struct sip_msg *msg);
 
```

Comment:
You should only need one version of each function as the functions are able to handle NULL pointers passed to them.

Reply:
Done. Thank you.